arches/settings_utils.py

```diff
@@ -40,20 +42,32 @@ def _move_to_end_of_sys_path(*paths, add_cwd=False):
         sys.path = _orig_sys_path
 
 
-def list_arches_app_names():
-    return [
-        config.name
-        for config in apps.get_app_configs()
-        if getattr(config, "is_arches_application", False)
-    ]
+def get_app_configs_from_module(module_name):
+    module = import_module(module_name)
+    app_configs = []
+    for app_name in getattr(module, "INSTALLED_APPS", []):
+        app_config = apps.get_app_config(app_name.split(".")[-1])
+        if getattr(app_config, "is_arches_application", False):
+            app_configs.append(app_config)
+    return app_configs
+
+
+def list_arches_app_names(application_name=None):
+    module_name = (
+        f"{application_name if application_name else settings.APP_NAME}.settings"
```

Comment:
I don't think we can depend on the settings module being called ".settings". This could cause subtle failures when:
- using test_settings
- using production_settings
- using docker_settings
etc

Reply:
I suppose we could document this requirement. If this is too difficult to deal with, is it sufficient to solve the underlying problem if we just leave these utils as they were and just copy the tsconfig files to the expected locations? If disco's config has 3 aliases, copying it to arches will have two extra aliases but hopefully no problem.

Actually, that's what seems to be happening anyway with site-packages, see comment.